plotly/figure_factory/_facet_grid.py

```diff
@@ -0,0 +1,787 @@
+    data = pd.DataFrame(data_list, columns=['column a', 'column b'])
+
+    row_list = []
+    col_list = []
+    for _ in range(tot_length):
+        row_list.append(random.choice(['apple', 'orange']))
+        col_list.append(random.choice(['dog', 'cat', 'bunny']))
+    data['fruit'] = row_list
+    data['animal'] = col_list
+
+    my_grid = ff.create_facet_grid(
+        data,
+        x='column a',
+        y='column b',
+        facet_row='fruit',
+        facet_col='animal'
+    )
+
+    py.iplot(my_grid, filename='facet_grid_row_and_col_facet')
+    ```
+
+    Example 5: Facet Grid with categorical color variable
+    ```
+    import plotly.plotly as py
+    import plotly.figure_factory as ff
+
+    import pandas as pd
+    import random
+
+    tot_length = 40
+    data_list = []
+    for _ in range(tot_length):
+        data_list.append([10*random.random() for _ in range(2)])
+    data = pd.DataFrame(data_list, columns=['column a', 'column b'])
+
+    row_list = []
+    col_list = []
+    for _ in range(tot_length):
+        row_list.append(random.choice(['apple', 'orange']))
+        col_list.append(random.choice(['dog', 'cat', 'bunny']))
+    data['fruit'] = row_list
+    data['animal'] = col_list
+
+    color_dict = {'dog': 'rgb(255,0,0)',
+                  'cat':'rgb(44,210,40)',
+                  'bunny':'rgb(0,0,255)'}
+    my_grid = ff.create_facet_grid(
+        data,
+        x='column a',
+        y='column b',
+        facet_row='fruit',
+        facet_col='animal',
+        color='animal',
+        color_dict=color_dict
+    )
+
+    py.iplot(my_grid, filename='facet_grid_cat_color')
+    ```
+
+    Example 6: Facet Grid with numerical color variable
+    ```
+    import plotly.plotly as py
+    import plotly.figure_factory as ff
+
+    import pandas as pd
+    import random
+
+    tot_length = 40
+    data_list = []
+    for _ in range(tot_length):
+        data_list.append([10*random.random() for _ in range(3)])
+    data = pd.DataFrame(data_list, columns=['column a',
+                                            'column b',
+                                            'column c'])
+
+    row_list = []
+    col_list = []
+    for _ in range(tot_length):
+        row_list.append(random.choice(['apple', 'orange']))
+        col_list.append(random.choice(['dog', 'cat', 'bunny']))
+    data['fruit'] = row_list
+    data['animal'] = col_list
+
+    color_dict = {'dog': 'rgb(255,0,0)',
+                  'cat':'rgb(44,210,40)',
+                  'bunny':'rgb(0,0,255)'}
+    my_grid = ff.create_facet_grid(
+        data,
+        x='column a',
+        y='column b',
+        facet_row='fruit',
+        facet_col='animal',
+        color='column c',
+        colorscale='Viridis'
+    )
+
+    py.iplot(my_grid, filename='facet_grid_numerical_color')
+    ```
+    """
+    if not pd:
+        raise exceptions.ImportError(
+            "'pandas' must be imported for this FigureFactory."
```

Comment:
since we changed the syntax I would probably say figure_factory or figure type or chart type